x-pack/solutions/security/plugins/elastic_assistant/server/__mocks__/mock_authenticated_user.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AuthenticatedUser } from '@kbn/core/server';
+
+export const mockAuthenticatedUser: AuthenticatedUser = {
```

Comment:
you could uses this one instead:
```
import { mockAuthenticatedUser } from '@kbn/core-security-common/src/authentication/authenticated_user.mock';
```

we also have one thats not as thorough here, maybe consider extending this one?
```
x-pack/solutions/security/plugins/elastic_assistant/server/__mocks__/user.ts
```

Reply:
thank you, will address in a follow-up